ksql-streams/src/main/java/io/confluent/ksql/execution/streams/TableGroupByBuilder.java

```diff
@@ -32,15 +33,14 @@
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.kstream.Grouped;
 import org.apache.kafka.streams.kstream.KGroupedTable;
-import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.KeyValueMapper;
 
 public final class TableGroupByBuilder {
   private TableGroupByBuilder() {
   }
 
   public static <K> KGroupedTable<Struct, GenericRow> build(
-      final KTable<K, GenericRow> ktable,
+      final KTableHolder<K> table,
```

Comment:
[nit] Could this be renamed to tableHolder? This goes for other holder variables in all the other execution plans. It seems weird to have table.getTable() and stream.getStream() (it looks like we already have a table available, why would we need to call getTable() from it?)